products/stratos/conf/data-bridge/thrift-client-config.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--Fields to be read and parsed into WSO2CEPStatisticsPublisher-->
+<thriftClientConfiguration>
+    <username>test User</username>
```

Comment:
@chamilad The values have been used temporarily to check the usage of a configuration file, in order to pass the necessary credentials required for WSO2CEPStatisticsPublisher. Next step is to make these values secure, therefore I also think these should be replaceable strings.